test/documentation/test_preprocess_documentation.t

```diff
@@ -0,0 +1,65 @@
+Testing interaction between `gospel pps` and Ocaml compiler. The issue is that
```

Comment:
Very nice test!
Doesn’t it miss some dune configuration to be run? I intentionally broke the test and `dune runtest` didn’t complain.